README.md

```diff
@@ -1,6 +1,6 @@
 # .NET Command Line Interface
 
-[![Join the chat at https://gitter.im/dotnet/cli](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dotnet/cli?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![.NET Slack Status](https://aspnetcoreslack.herokuapp.com/badge.svg?2)](http://tattoocoder.com/aspnet-slack-sign-up/) [![Join the chat at https://gitter.im/dotnet/cli](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dotnet/cli?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
That takes it to the slack page an has no sign up or signup customisation so looks like:

![slack-signup](http://aoa.blob.core.windows.net/aspnet/slack-signup.png)

Where as the link allows new people to get signed in